api_confluence.py

```diff
@@ -37,6 +46,56 @@ def url_page(page_id):
 def url_attachments(page_id):
     return f"{URL_WIKI_REST_API}/content/{page_id}/child/attachment"
 
+# ------------------------------------------------------------------
```

Comment:
Adding section comment / dividers for clarity.  Module will now be processing 2 different kinds of Confluence pages
- Looker graphs (all have same format & YAML configs)
- Custom reports (all have different format & use XML templates)